lib/helpers/is-valid-config-object.js

```diff
@@ -0,0 +1,34 @@
+export default function isValidConfigObjectFormat(config, POSSIBLE_CONFIG) {
```

Comment:
Can we add some unit tests for this function (in a corresponding `is-valid-config-object-test.js` file alongside other helper unit tests)? Should be easy enough to test.

Reply:
Tests have been added!